### PR TITLE
chore(tests): add menu tests

### PR DESCRIPTION
**Pull request by hypeJunction**

### CODE REVIEW

File: engine/tests/phpunit/unit/Elgg/Navigation/MenuRenderingTest.php [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+		$this->user = $this->createUser();
+		elgg_set_page_owner_guid($this->user->guid);
+		$this->object = $this->createObject();
+		$this->group = $this->createGroup();
+
+		elgg_trigger_event('init', 'system');
+	}
+
+	public function down() {
+
```
Comment (jdalsem) [inline engine/tests/phpunit/unit/Elgg/Navigation/MenuRenderingTest.php]: why not cleanup what you create in up()?

**Comment by jeabakker** [comment]
wouldn't this be an integration test instead of an unit test? Otherwise OK

**Comment by hypeJunction** [comment]
Integration tests are expensive and unit tests are pretty good these days

File: engine/tests/phpunit/unit/Elgg/Navigation/MenuRenderingTest.php (hunk 1 above)
Comment (hypeJunction) [reply engine/tests/phpunit/unit/Elgg/Navigation/MenuRenderingTest.php]: Parent class cleans up db query specs

**Pull request merged**